locales/en.json

```diff
@@ -5,7 +5,8 @@
       "message": "We have reported this to our team! Please close the app and start again!"
     },
     "app":  {
-      "version": "Version"
+      "version": "Version",
+      "title": "ItaliaApp"
```

Comment:
we should probably use [this](https://github.com/rebeccahughes/react-native-device-info#getapplicationname)

Reply:
Fixed.